package.json

```diff
@@ -1,51 +1,35 @@
 {
-  "name": "rln-contract",
-  "license": "(MIT OR Apache-2.0)",
-  "scripts": {
-    "start": "hardhat node --export-all deployments/allDeployments.json",
-    "compile": "hardhat compile",
-    "test": "yarn test:foundry && yarn test:hardhat",
-    "test:verbose": "yarn test:foundry -vvv && yarn test:hardhat --verbose",
-    "test:hardhat": "hardhat test",
-    "test:hardhat:localhost": "yarn test:hardhat --network localhost",
-    "test:hardhat:sepolia": "yarn test:hardhat --network sepolia",
-    "test:foundry": "forge test",
-    "deploy": "hardhat deploy --export-all deployments/allDeployments.json --network",
-    "deploy:sepolia": "yarn deploy sepolia",
-    "deploy:localhost": "yarn deploy localhost",
-    "verify:sepolia": "hardhat --network sepolia etherscan-verify",
-    "coverage": "forge coverage --report lcov",
-    "fmt": "prettier --write \"**/*.{js,ts}\"",
-    "lint": "prettier --check \"**/*.{js,ts}\"",
-    "prepare": "husky install",
-    "docgen": "hardhat docgen"
+  "name": "@vacp2p/foundry-template",
+  "description": "Foundry-based template for developing Solidity smart contracts used by Vac",
+  "version": "1.0.0",
+  "author": {
+    "name": "0x-r4bbit",
+    "url": "https://github.com/vacp2p"
```

Comment:
This part probably needs change (name, description, author)

Reply:
Addressed in a647a84